test/htmlwriter.jl

```diff
@@ -296,6 +315,11 @@ end
         @test html.size_threshold_warn == 1234
     end
 
+    @testset "HTML: size_threshold_result_formats" begin
+        @test typeof(format_units(1024)) == String
+        @test typeof(format_units(typemax(Int))) == String
```

Comment:
Slightly expanding, to cover more edge cases, and also making sure (1) we test the actual function, and (2) actually testing the output (which is the interesting thing to test for):

```suggestion
        @test HTMLWriter.format_units(0) == "0 (bytes)"
        @test HTMLWriter.format_units(1) == "1 (bytes)"
        @test HTMLWriter.format_units(1023) == "1023 (bytes)"
        @test HTMLWriter.format_units(1024) == "1 (KiB)"
        @test HTMLWriter.format_units(4000) == "4 (KiB)"
        @test HTMLWriter.format_units(2^20 + 123) == "@ (MiB)"
        @test HTMLWriter.format_units(typemax(Int) == "(no limit)"
```

Not 100% sure if these tests pass actually.